<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySudoku</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SudokuView" name="tableView"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="helpLayout">
      <item>
       <widget class="QGroupBox" name="filterGroupBox">
        <property name="title">
         <string>Filters</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="directFilterButton">
           <property name="text">
            <string>Direct</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="indirectFilterButton">
           <property name="text">
            <string>Indirect</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="groupFilterButton">
           <property name="text">
            <string>Group</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="hiddenFilterButton">
           <property name="text">
            <string>HiddenGroup</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="noChoiceFilterButton">
           <property name="text">
            <string>NoChoice</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="helperGroupBox">
        <property name="title">
         <string>Shortcut</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="autofillButton">
           <property name="text">
            <string>Autofill</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="solveButton">
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoader">
    <property name="title">
     <string>Initialize Sudoku</string>
    </property>
    <addaction name="actionInitManually"/>
    <addaction name="actionLoadFromCsv"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionAutocheck"/>
    <addaction name="actionDisplayHelp"/>
   </widget>
   <addaction name="menuLoader"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionLoadFromCsv">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load from csv</string>
   </property>
  </action>
  <action name="actionAutocheck">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Autocheck</string>
   </property>
  </action>
  <action name="actionDisplayHelp">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Display help</string>
   </property>
  </action>
  <action name="actionInitManually">
   <property name="text">
    <string>Init. manually</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SudokuView</class>
   <extends>QTableView</extends>
   <header>sudokuView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionDisplayHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>filterGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDisplayHelp</sender>
   <signal>toggled(bool)</signal>
   <receiver>helperGroupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>742</x>
     <y>236</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
